src/convertToObject.js

```diff
@@ -1,18 +1,22 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesTab = sourceString.split(';').filter(el => el.trim() !== '');
```

Comment:
According to the checklist:
_All chained methods called on the same level should have same indentation. This makes code much more readable._
```suggestion
  const stylesTab = sourceString
  .split(';')
  .filter(el => el.trim() !== '');
```

Also I would name this variable **stylesArr** instead of **stylesTab** (as array is more intuitive)
It would be also good to find better and more descriptive name for **el**